Libraries/D3D12RaytracingFallback/src/UberShaderRayTracingProgram.cpp

```diff
@@ -81,8 +85,6 @@ namespace FallbackLayer
             m_DxilShaderPatcher.RenameAndLink(libraryInfo, stateObjectCollection.m_exportDescs, &pAppLibrariesBlob);
         }
 
-        
-
         std::vector<DxilLibraryInfo> librariesInfo;
         DxilLibraryInfo outputLibInfo((void *)pAppLibrariesBlob->GetBufferPointer(), (UINT)pAppLibrariesBlob->GetBufferSize());
         CComPtr<IDxcBlob> pOutputBlob;
```

Comment:
General - non-blocking: We shouldn't be using ATL CComPtr's in 2018. Consider using WRL::ComPtr instead

Reply:
+1, but FL uses CComPtr all over the place so there's something to be said about consistency for now.

Reply:
Ugh